integration-tests/src/test/java/oracle/kubernetes/operator/ItModelInImageConfigUpdate.java

```diff
@@ -425,6 +425,179 @@ public void testMiiConfigUpdateJdbcImage() {
     LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethodName);
   }
 
+  /**
+   * Create a domain without a JDBC DS using model in image and having configmap
+   * in the domain.yaml before deploying the domain. After deploying the domain crd,
```

Comment:
The CRD is created by the operator.  The test will create a domain resource.

Reply:
k. let me change

Reply:
changed to After the domain resources is created,